testing/ef-tests/tests/serialization.rs

```diff
@@ -0,0 +1,330 @@
+          "logs_bloom": "0x2102046181001b0000148e5400043546408135004811410800884040409080020034a000008200801841024088843128060008008800204400108222486004488a00e82000c201395801421880206020000d000412500c00800000058202040052c404001200125102030206211208180d0008c82880361090c04010480ba4420208002802224084100880140b1521160241880147201202009a8004c834b8081200228109922802086900a1400380120288821880080050000804e0028003420201c60208640840850a402a1054088480c40004400c02028820370a202022400830a1482204040809000210004089438445000058c04241520b000208400000",
+          "prev_randao": "0x937766d58929b5ee4ef89d49f31ef0eaa890105b5aba76b030419a35f57ffceb",
+          "block_number": "22315643",
+          "gas_limit": "35999895",
+          "gas_used": "5002300",
+          "timestamp": "1745217623",
+          "extra_data": "0xe29ca82051756173617220287175617361722e77696e2920e29ca8",
+          "base_fee_per_gas": "346220230",
+          "block_hash": "0x1d3e3eb61ef699c48d3a05b5834f07de629ad5090a91589bde981fac0db65674",
+          "transactions": [
+            "0x02f901520182242284aa78230084aa7823008302d6699468b3465833fb72a70ecdf485e0e4c7bd8665fc4580b8e404e45aaf000000000000000000000000168e209d7b2f58f1f24b8ae7b7d35e662bbf11cc000000000000000000000000dac17f958d2ee523a2206206994597c13d831ec70000000000000000000000000000000000000000000000000000000000002710000000000000000000000000bbb5008da7ef90a416a4389fdad3872d6896cff900000000000000000000000000000000000000000000102b859769ac0ae600000000000000000000000000000000000000000000000000000000000006d3fda90000000000000000000000000000000000000000000000000000000000000000c080a047349d2a344a6d10319f5bbec1663cb446a0dc20c788e84c417dda4c8c3bb028a0398ab564931aca0bc85cae14ff2b92e2b2322274d5a0074d1054b5ae056b300d",
+            "0x02f8d2018304a5168459682f008483cf87fe82f2089446950ba8946d7be4594399bcf203fb53e1fd7d3780b8648f975a6400000000000000000000000064bc2ca1be492be7185faa2c8835d9b824c8a194000000000000000000000000410f4d628cfc6e788b54206022cc01341d6166d800000000000000000000000000000000000000000000000a7669bd7132640000c080a00bfb617f48517a2afddeaebfa67d88e55cce68b46ef5f329c1e5416ebc876a80a04d8b9eceeb4858489f5792a56287f6e14f463c40096497687f50101064622b92"
+          ],
+          "withdrawals": [
+            {
+              "index": "84490907",
+              "validator_index": "253071",
+              "address": "0x9f4cf329f4cf376b7aded854d6054859dd102a2a",
+              "amount": "19091632"
+            }
+          ],
+          "blob_gas_used": "786432",
+          "excess_blob_gas": "63700992"
+        },
+        "bls_to_execution_changes": [],
+        "blob_kzg_commitments": [
+          "0x82fb14e2d1c4262498be8c164219ebc4949dbbb8bd7b626fac4422a4408b20569989f4579367ef6433a7d83d0addf800"
+        ]
+      }
+    },
+    "signature": "0x96e3ac0f837f8210b1eb8caff790eef4111d0e28ef1d1600e418c1451916340e88b8176a864f64a7decdf13c839e677b04d3ebb219c21b95c3e57f1f55f7c097f07736b7c8961e3fee9574793f8329494b8902143fa6b1de3dd1717178e1f506"
+  }
+}
+"#;
```

Comment:
can you make a test assets folder in the `tests` folder, then place these into 2 seperate json files. Then read these jsons files into the test instead of using static